docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-virtual-background      | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background)           |
 
 
+## 2024-09-30
+Released: `@100mslive/hms-video-store@0.12.21`, `@100mslive/react-sdk@0.10.21`, `@100mslive/hls-player@0.3.21`, `@100mslive/roomkit-react@0.3.21`, `@100mslive/hms-whiteboard@0.0.11`, `@100mslive/hms-virtual-background@1.13.21`
+
+### Fixed:
+- Roomkit Prebuilt: HLS player takes more space in fullscreen mode
+- Roomkit Prebuilt: Device label categorization
```

Comment:
this is broken I think. Also the message should be just the icon not showing correct device. this does not tell anything